scripts/webpack/webpack.config.e2e.ts

```diff
@@ -9,16 +9,14 @@ const { paths } = config;
 const webpackConfig: webpack.Configuration = {
   name: 'client',
   target: 'web',
-  mode: config.compiler_mode,
+  mode: 'production',
   entry: {
     app: paths.e2eSrc('app'),
   },
   output: {
     filename: `[name].js`,
     path: paths.e2eDist(),
-    pathinfo: true,
   },
```

Comment:
```suggestion
  },
  // CI should not use sourcemaps, but it's useful for local debugging
  devtool: process.env.CI ? false : config.compiler_devtool,
```

I suggest to restore it for local builds.
**Not sure** that "process.env.CI" is a valid variable, please check, first.